### ISSUE TITLE
[WINDOWS] WindowsError: [Error 2] Nie mo�na odnale�� okre�lonego pliku

### ISSUE THREAD

**Issue by pawelarybak** [LekSeek/django-e2e-runner#1]
Pod windowsem cypress się nie odpala, ponieważ w następującej linijce:

https://github.com/LekSeek/django-e2e-runner/blob/f52514c904cd0a170199706d998e3da0e2523e54/django_e2e_runner/settings.py#L19

są używane unixowe slashe (pod windowsem powinny być backslashe i to podwójne). Z tego powodu lepiej używać os.path.join zamiast hardkodować ścieżki ze slashami.

**Comment by olafik** [comment]
Tak, to jedna z wielu rzeczy występujących obecnie w kodzie, które są zrobione tymczasowo i na pewno są do zmiany – ścieżka tu podana ewidentnie jest zrobiona pod jakiś konkretny projekt.

Aktualnie na liście naszych priorytetów znajdują się inne rzeczy i nie jestem w stanie powiedzieć, kiedy zajmiemy się tym i innymi drobiazgami. Zapraszamy użytkowników do otwierania pull requestów (próśb o wciąganie), jeżeli chcą pomóc w szybszym wdrożeniu danej zmiany (a w szerszym kontekście: w szybszym wypuszczeniu stabilnej wersji django-e2e-runner).

W tym konkretnym przypadku rozwiązać problem można też w ten sam sposób, który stosowałoby się w projekcie, gdzie ścieżka do Cypressa jest inna niż wyżej podana: w pliku settings.py w Twoim projekcie djangowym powinieneś ustawić wartość E2E_TEST_RUNNER_EXECUTABLE na odpowiednią dla Twojego projektu. To zachowanie nie jest jeszcze opisane w dokumentacji, ale plik django_e2e_runner/settings.py przechowuje jedynie wartości domyślne, które pod każdym projektem djangowym można zmienić na własne.

Mimo powyższego, zapytam z ciekawości w jakim konkretnie środowisku wystąpił ten błąd? Mieliśmy okazję używać ten kod na dwóch instancjach Windowsa 10 i podany problem nie występował.
